### PR TITLE
Remove "Read vars_file" for every playbook

### LINKED ISSUE #409: [New Feature] Too many "Read vars_file '{{ testing_vars_file | default('../vars/test.yml') }}'"
##### SUMMARY

We need to find some way to reduce multiple times reading vars file:
```
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
```

We can put below task into main.yml to see whether it can improve.
ansible.builtin.include_vars:
    file: "{{ testing_vars_file | default('../vars/test.yml') }}"

##### ISSUE TYPE
- Feature Idea

##### ENVIRONMENT / OS

Linux/Windows

### PR DESCRIPTION
**Pull request by Winnie81**
Fix #409  Remove "Read vars_file" for Linux vm deployment and test passed.

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@
 - name: main
   hosts: localhost
   tasks:
+    - name: "Read variables from vars file"
+      ansible.builtin.include_vars:
+        file: "{{ testing_vars_file | default('../vars/test.yml') }}"
```
Comment (Tomorrow9) [inline main.yml:10]: '../vars/test.yml' -> 'vars/test.yml'

File: windows/mdag_enable_disable/mdag_enable_disable.yml [hunk 2]
```diff
@@ -62,4 +60,4 @@
           include_tasks: disable_mdag.yml
       rescue:
         - name: "Test case failure"
-          include_tasks: ../../common/test_rescue.yml
\ No newline at end of file
+          include_tasks: ../../common/test_rescue.yml
```
Comment (Tomorrow9) [inline windows/mdag_enable_disable/mdag_enable_disable.yml:63]: Remove empty line.

File: windows/wsl_distro_install_uninstall/wsl_distro_install_uninstall.yml [hunk 3]
```diff
@@ -39,4 +37,4 @@
         # Uninstall the distribution
         - include_tasks: unregister_wsl_distro.yml
       rescue:
-        - include_tasks: ../../common/test_rescue.yml
\ No newline at end of file
+        - include_tasks: ../../common/test_rescue.yml
```
Comment (Tomorrow9) [inline windows/wsl_distro_install_uninstall/wsl_distro_install_uninstall.yml:40]: Remove empty line.

**Review by keirazhang — APPROVED** [review]

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**